src/main/authentication/Listener/Platform/OauthSsoListener.php

```diff
@@ -12,23 +13,27 @@ class OauthSsoListener
     private $config;
     /** @var OauthManager */
     private $oauthManager;
+    /** @var AuthenticationManager */
+    private $authenticationManager;
 
     public function __construct(
         PlatformConfigurationHandler $config,
-        OauthManager $oauthManager
+        OauthManager $oauthManager,
+        AuthenticationManager $authenticationManager
     ) {
         $this->config = $config;
         $this->oauthManager = $oauthManager;
+        $this->authenticationManager = $authenticationManager;
     }
 
     public function onConfig(GenericDataEvent $event)
     {
         $event->setResponse([
             'authentication' => [
-                'help' => $this->config->getParameter('authentication.help'),
-                'changePassword' => $this->config->getParameter('authentication.changePassword'),
-                'internalAccount' => $this->config->getParameter('authentication.internalAccount'),
-                'showClientIp' => $this->config->getParameter('authentication.showClientIp'),
+                'help' => $this->authenticationManager->getParameters()->getHelpMessage(),
```

Comment:
Stocke le résultat de `$this->authenticationManager->getParameters()` dans une variable au lieu de l'appeler à chaque param.